README.md

```diff
@@ -4,19 +4,22 @@ Transit ITS Data Exchange Specification (TIDES) for historical transit operation
 
 This repository provides data schemas and tools to support the access, management, and improvement of historical transit operations data, including vehicle operations, passenger activity, fare collection, and other similar data.
 
-### TIDES specification
-The TIDES specification is maintained in the `/spec` sub-directory as a series of JSON tables.
+## TIDES specification
 
-Data Table schemas
-Data table schemas are specified in JSON and are compatible with the [frictionless data](https://specs.frictionlessdata.io/table-schema/) table schema standards.
+The TIDES specification is maintained in the `/spec` sub-directory as a series of tables specified as a [frictionless data package](https://specs.frictionlessdata.io/data-package/) defined in `/spec/tides.spec.json`, which is a series of [frictionless data tables](https://specs.frictionlessdata.io/table-schema/) schemas specified in the set of files `/spec/*.schema.json`.
+
+Human readable tables describing the schema can be found at [http://tides-transit/github.io/TIDES/architecture](http://tides-transit/github.io/TIDES/architecture).
```

Comment:
Is this url correct? I think it should be `<http://tides-transit.github.io/TIDES/architecture>`

Reply:
I feel like a doofus, but isn't that what that URL is (in [Markdown format](https://www.markdownguide.org/basic-syntax/#links))

Reply:
or are you saying to just use the shortcut with < >?

Reply:
Oh, no, there appears to be a `/` instead of a `.`
```
http://tides-transit/github.io/TIDES/architecture
                    ^
```

Reply:
🤦‍♀️

Reply:
👓